packages/ton/package.json

```
@@ -0,0 +1,64 @@
+{
+  "name": "@ant-design/web3-ton",
+  "version": "1.0.0",
```

这里是 `1.0.0` 的话，再发一个 major 是不是会变成 2.0.0 @jeasonstudio 这里估计要改为 0.x

```suggestion
  "version": "0.0.0",
```